tutornelc_extensions/patches/openedx-common-settings

```diff
@@ -0,0 +1,3 @@
+{% if NELC_EXTENSIONS_MYSQL_SSL_CA_CONFIG %}
```

Comment:
this condition has been already included into the patch, so in my opinion this is redundant, same comment for other patches

Reply:
[b83d873](https://github.com/nelc/tutor-contrib-nelc-extensions/pull/11/commits/b83d8739a2cc6f208ffdd8335302cf903e1441d5)'